models/workitemtype.go

```diff
@@ -90,14 +90,15 @@ func (wit WorkItemType) Equal(u convert.Equaler) bool {
 }
 
 // ConvertFromModel serializes a database persisted workitem.
-func (wiType WorkItemType) ConvertFromModel(workItem WorkItem) (*app.WorkItem, error) {
+func (wit WorkItemType) ConvertFromModel(workItem WorkItem) (*app.WorkItem, error) {
 	result := app.WorkItem{
 		ID:      strconv.FormatUint(workItem.ID, 10),
 		Type:    workItem.Type,
 		Version: workItem.Version,
 		Fields:  map[string]interface{}{}}
 
-	for name, field := range wiType.Fields {
+	result.Fields["created_at"] = workItem.CreatedAt
```

Comment:
I have added a constant named `SystemCreatedAt` with value as `created_at`. But do we really need that `System` prefix?